blocks/hotspot-research/hotspot-research.js

```diff
@@ -0,0 +1,23 @@
+// media query match that indicates mobile/tablet width
+const isDesktop = window.matchMedia('(min-width: 900px)');
```

Comment:
You can get the `deviceType` [from here](https://github.com/sunstar-global/sunstar/blob/b82322d538c4c9dac391a0a75c9ce917605f283a/scripts/lib-franklin.js#L820)